modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase

```diff
@@ -26,29 +26,26 @@ definition {
 		}
 
 		task ("Employee user creates an organization") {
```

Comment:
Other type of user can create an organization or just an employee user? If it's just an employee, maybe change the task description to "Create an organization" or something similar.

Reply:
Others users can create, but following the business role, the employee is the user that starts the flow, creating an organization. Because a staff can create due to they are also employees. So on our test we are creating organization or request with employee I will change  this, thanks for point this